otx/algorithms/action/tasks/train.py

```diff
@@ -120,7 +120,7 @@ def train(
         else:
             update_progress_callback = default_progress_callback
         self._time_monitor = TrainingProgressCallback(update_progress_callback)
-        self._learning_curves = DefaultDict(OTXLoggerHook.Curve)
+        self._learning_curves = UncopiableDefaultDict(OTXLoggerHook.Curve)
```

Comment:
Is this needed even if `self._learning_curves` is initialized in `__init__()` in `BaseTask`?

Reply:
I fixed it. please check again.